en-US/Players_Guide/03_Creating_a_Character/Cultures/Cultures/Nomad.md

```diff
@@ -1,4 +1,19 @@
 #### Nomad
 
-> **Warning:**
-> This culture has not yet been transferred from the Google Doc.
+Wanderers and explorers, this culture values the journey over the destination. Its members are resourceful, attuned to nature, and skilled at navigating physical and social challenges.
+\
+\
+**Nature Minded.**
+You are proficient in Animal Handling and Survival, and either Medicine, Nature, or Perception.
+\
+**On the Road Again.**
+You are proficient with land vehicles and tinker’s tools.
+You can use tinker’s tools to repair a land vehicle you have traveled in for at least 1 month, and gain expertise on any checks made to do so.
```

Comment:
I changed "travelled" to "traveled", since this is US English rather than British English.